### ISSUE TITLE
docker image appears to have two numpy versions installed on top of each other

### ISSUE THREAD

**Issue by vladium** [openvinotoolkit/openvino#831]
- this applies to versions 2020.2 and 2020.3
```
>docker run -it --rm openvino/ubuntu18_dev bash -l
openvino@dd99ad959eb3:/opt/intel/openvino_2020.3.194$ l /usr/local/lib/python3.6/dist-packages/ | grep numpy
numpy/
numpy-1.16.2.dist-info/
numpy-1.17.5-py3.6-linux-x86_64.egg/
```
It seems to mostly work fine, but in some containers "import numpy" triggers an import failure
```
ImportError: Something is wrong with the numpy installation. While importing we detected an older version of numpy in ...
```

**Comment by yury-gorbachev** [comment]
@moslex might be interesting for you

**Comment by 0994210227** [comment]
#784

**Comment by generalova-kate** [comment]
Fixed in 2021.1 docker images
